db/seeds.rb

```diff
@@ -1,7 +1,21 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'faker'
+include Faker
+
+1.times do
+  User.create!(
+    email: Faker::Internet.email,
+    password: Faker::Internet.password
+    )
+end
+
+10.times do
+  Item.create!(
+   user: @user_id,
```

Comment:
I've tried a couple of different syntaxes to get `user_id` in there, this one didn't give me an error, but it still didn't assign the item's to the user.  This is what the curriculum would lead me to believe is the right answer.